d2bs/kolbot/libs/common/Attack.js

```diff
@@ -233,6 +233,8 @@ var Attack = {
 
 		if (Config.MFLeader) {
 			Pather.makePortal();
+			Pather.TeamCheck(1);    //add by WhyY0416
```

Comment:
your changes are annotated in git, so your name is already on these lines as far as git is concerned. You don't need these comments with your name :)